app/src/main/java/fr/neamar/kiss/broadcast/InstallShortcutHandler.java

```diff
@@ -61,7 +61,7 @@ public void onReceive(Context context, Intent data) {
             }
         }
                 
-        dh.getShortcutProvider().addShortcut(pojo);
+        dh.addShortcut(pojo);
```

Comment:
Ok, I understand, but still it feels like a breach in our separation of concern.

Maybe it's better to either:
* always start shortcut services, but return no pojo when it's disabled
* check whether the shortcut service is running, and if it's not just skip the shortcut creation (after all, shortcuts are disabled)